sound/usb/quirks.c

```diff
@@ -1114,7 +1114,8 @@ static void set_format_emu_quirk(struct snd_usb_substream *subs,
 	 * by playback substream
 	 */
 	if (subs->direction == SNDRV_PCM_STREAM_PLAYBACK) {
-		if (subs->stream->substream[SNDRV_PCM_STREAM_CAPTURE].interface != -1)
+		if (subs->stream->substream[SNDRV_PCM_STREAM_CAPTURE].interface != -1 &&
```

Comment:
i think it would be better if you could check the index of the array before access it (save array access)
so you need to check if `subs->stream->substream[SNDRV_PCM_STREAM_CAPTURE]` is not `NULL`